Assembly-CSharp/ReflectionHelper.cs

```diff
@@ -169,6 +169,10 @@ public static FieldInfo GetFieldInfo(Type t, string field, bool instance = true)
 
             gen.Emit(OpCodes.Ldarg_0);
             gen.Emit(OpCodes.Ldfld, fi);
+            if (fi.FieldType.IsSubclassOf(typeof(ValueType)) && typeof(TField) == typeof(object))
```

Comment:
Why not `.IsValueType`?

Reply:
also nit - would prefer this w/out the braces and one line whitespace separated on both sides but that's w/e

Reply:
> Why not `.IsValueType`?

Because I didn't know that existed...